clang/test/Driver/flang/dev_null_as_input.f95

```diff
@@ -0,0 +1,10 @@
+! REQUIRES: classic_flang
+
+! Check that the driver invokes flang1 correctly for preprocessed fixed-form
+! Fortran code.
+
+! RUN: %clang --driver-mode=flang -E -target x86_64-unknown-linux-gnu -c - </dev/null -### 2>&1 \
```

Comment:
This test may need `REQUIRE: linux` for the use of `/dev/null` in the `RUN` command, in which case we probably should also have a Windows version of the test.

Reply:
@bryanpkc What do You mean? Could you give an example?

Reply:
I meant that you should probably tell `llvm-lit` not to run this test if the current operating system is not Linux. But I think the correct feature tag to use is `system-linux`, not `linux`. Some examples are:

- https://github.com/llvm/llvm-project/blob/main/clang/test/Analysis/ctu-on-demand-parsing.c#L22
- https://github.com/llvm/llvm-project/blob/main/clang/test/Driver/amdgpu-openmp-system-arch.c#L1

Reply:
@pwisniewskimobica this still needs to be changed.

Reply:
@pawosm-arm 
Just fixed. Thanks

Reply:
Since there is now Windows support for the same feature, please provide a Windows version of this test.

Reply:
@bryanpkc Instead of one test file I just created separate for windows and linux. Please take a look. Thanks